dev/docker-compose.dev.yml

```diff
@@ -15,7 +15,6 @@ services:
             - ../openslides_backend:/app/openslides_backend
             - ../tests:/app/tests
             - ../cli:/app/cli
-            - ../global:/app/global
             - ../scripts:/app/scripts
         environment:
```

Comment:
I would think, that the data and meta are missing in the docker compose part now.
Pls test.

Reply:
Yes you are right. Everything was working but it should be included into the volumes. As well should requirements; to be more consistent.